lib/foreman_remote_execution/engine.rb

```diff
@@ -134,6 +135,9 @@ class Engine < ::Rails::Engine
           ctx.permit :execution
         end
 
+        extend_template_helpers ForemanRemoteExecution::RendererMethods
+        allowed_template_helpers :find_job_invocation_by_id, :load_job_invocations, :search_in_hosts_relation
```

Comment:
The `search_in_hosts_relation` should be enabled in the core PR, right?
```suggestion
        allowed_template_helpers :find_job_invocation_by_id, :load_job_invocations
```